Client/game_sa/CFontSA.h

```diff
@@ -0,0 +1,141 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        game_sa/CFontSA.h
+ *  PURPOSE:     Header file for font class
+ *
+ *  Multi Theft Auto is available from https://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#pragma once
+#include <CVector2D.h>
+#include <game/RenderWare.h>
+#include <game/CHud.h>
+
+#define FUNC_CFont_PrintChar             0x718A10
+#define FUNC_CFont_PrintString           0x71A700
+#define FUNC_CFont_PrintStringFromBottom 0x71A820
```

Comment:
If you're not going to use these constants in any other file, then just remove all of them, and use the function address directly in the calling lines.

Reply:
Won't this go against the coding guidelines? I heard we are supposed to avoid 'magic values.'

Reply:
That's correct, but you can also move that named constant right above the calling line, if you don't use that constant anywhere else.